.buildbaselog

```diff
@@ -9,4 +9,7 @@
 -   Add your reason here...
 
 *   Jul 15 2021 <danfengl@vmware.com>
--   Create this file to trigger build base action in buld-package workflow 
\ No newline at end of file
+-   Create this file to trigger build base action in buld-package workflow 
+
+*   Sept 22 2021 <jiaoya@vmware.com>
```

Comment:
Logs should be listed by creation time in descending order.

Reply:
solved